exchange/adapter_map.go

```diff
@@ -69,11 +69,18 @@ func newAdapterMap(client *http.Client, cfg *config.Configuration, infos adapter
 	}
 
 	allBidders := make(map[openrtb_ext.BidderName]adaptedBidder, len(ortbBidders)+len(legacyBidders))
+
 	for name, bidder := range legacyBidders {
-		allBidders[name] = adaptLegacyAdapter(bidder)
+		// Clean out any disabled bidders, or bidders that are otherwise missing infos (Only reason for not having infos should be that they are disabled)
```

Comment:
re: my tech talk... this is a great example of a confusing Message.

1. If they see the name & type "BidderInfos," why would either a caller or callee think "enabled bidders"?
2. The `cfg` is already a function argument, and it has `Disabled` info in it very clearly.

Better to read from the config here.

Reply:
My hack saved a lot of headache. Apparently we have several tests that depend on `newAdapaterMap()` working given a blank config. Cleaning up the mess that results when enforcing that `cfg.Adapters[<name>].Disabled` exist and be false. Should hopefully improve our code to be a bit more strict. Surprised that the function works at all on a blank config.